Vic2ToHoI4/Data_Files/blankMod/output/common/modifiers/00_static_modifiers.txt

```diff
@@ -0,0 +1,535 @@
+	naval_retreat_speed = 0.05
+	positioning = -0.1
+}
+weather_blizzard = { # on Region 
+	air_detection = -0.3
+	naval_detection = -0.3
+	naval_hit_chance = -0.1
+	naval_speed_factor = -0.1
+	carrier_traffic = -1.0
+	naval_strike = -0.4
+	air_accidents = 0.3
+	air_bombing_targetting = -0.7
+	local_org_regain = -0.3
+	air_mission_efficiency = -0.3
+	naval_retreat_speed = 0.15
+	positioning = -0.2
+}
+weather_sandstorm = { # on Region 
+	air_detection = -0.9
+	air_accidents = 0.6
+	naval_strike = -0.9
+	air_bombing_targetting = -1
+	local_org_regain = -0.5
+	air_mission_efficiency = -0.5
+}
+
+weather_arctic_water = { # on Region 
+	navy_casualty_on_sink = 0.2
+	naval_attrition = 0.1
+	naval_retreat_speed = -0.05
+	positioning = -0.2
+}
+
+weather_mud = { # on Province
+	attrition = 0.7
+	army_speed_factor = -0.5
+	army_attack_factor = -0.4
+}
+weather_extreme_cold = { # on Province
+	winter_attrition = 0.2
+	dig_in_speed_factor = -0.6
+	local_org_regain = -0.05
+	army_attack_factor = -0.2
+}
+weather_very_cold = { # on Province
+	winter_attrition = 0.1
+	dig_in_speed_factor = -0.3
+	army_attack_factor = -0.1
+}
+weather_very_hot = { # on Province
+	heat_attrition = 0.1
+	local_org_regain = -0.05
+}
+weather_extreme_hot = { # on Province
+	heat_attrition = 0.2
+	local_org_regain = -0.1
+	supply_consumption_factor = 0.5
+	army_attack_factor = -0.1
+}
+weather_ground_snow_medium = { # on Province
+	army_speed_factor = -0.1
+	army_attack_factor = -0.1
+}
+weather_ground_snow_high = { # on Province
+	army_speed_factor = -0.25
+	local_org_regain = -0.05
+	army_attack_factor = -0.3
+	naval_retreat_speed = 0.1
+}
+
+flooded = { # on Province
+	army_speed_factor = -0.5
+	army_defence_factor = 0.5
+	dig_in_speed_factor = 0.2
+}
+
+unplanned_offensive = { # on Province
+	army_speed_factor = -0.75
+	army_attack_factor = -0.9
+	local_org_regain = -0.5
+}
+
+night = { # On province. Multiplied by amount of darkness.
+	naval_hit_chance = -0.25
+	carrier_traffic = -1.0
+	air_bombing_targetting = -0.5
+	naval_retreat_speed = 0.1
+}
+
+# The following is multiplied by local resistance strength.
+resistance_effect_base = {
+	# todo - spy defense
+}
+
+resistance_effect = {
+	#local_supplies = -0.5
+	#local_intel_to_enemies = 10
+	#local_factory_sabotage = 0.75
+	#attrition = 0.25
+}
+
+# For compliance base + effect * compliance_level will be applied as a modifier.
+compliance_effect_base = {
+	#local_factories = -0.75
+	#local_manpower = -0.75
+	#local_resources = -0.75
+}
+
+compliance_effect = {
+	local_factories = 0.65
+	local_non_core_manpower = 0.18
+	local_resources = 0.60
+}
+
+# On States that are owned by a non-core country
+non_core = {
+}
+
+# On States that are controlled by a non-core country
+non_core_controller = {
+	local_building_slots_factor = -0.5
+	
+	local_factories = -0.75
+	#local_manpower = -0.75
+	local_resources = -0.65
+}
+
+lacking_consumer_goods = {
+}
+
+gain_focus = {
+	political_power_cost = 1
+} 
+
+#license production
```

Comment:
Why should we keep country specific license opinions?

Reply:
I was aiming to just remove items that would clearly be problematic. I think it's just the name that has a tag here, so it shouldn't cause problems itself.